docker-compose.yml

```diff
@@ -48,7 +48,7 @@ services:
     image: 618209257270.dkr.ecr.us-west-2.amazonaws.com/nics/proxy:latest
     ports:
       - "3000:80"
```

Comment:
I suggest you also move http to 80 rather than 3000 if you're going to use 443 for https.